jobs/config.json

```diff
@@ -8506,6 +8506,27 @@
       "sig-cluster-lifecycle"
     ]
   },
+  "ci-kubernetes-e2e-kubeadm-gce-ipvs": {
+    "args": [
+      "--cluster=",
+      "--deployment=kubernetes-anywhere",
+      "--env-file=jobs/platform/gce.env",
+      "--extract=ci/latest",
+      "--gcp-zone=us-central1-f",
+      "--kubeadm=ci",
+      "--kubernetes-anywhere-dump-cluster-logs=true",
+      "--kubernetes-anywhere-kubelet-ci-version=latest",
+      "--kubernetes-anywhere-kubernetes-version=ci/latest",
+      "--kubernetes-anywhere-proxy-mode=ipvs",
+      "--provider=kubernetes-anywhere",
+      "--test_args=--ginkgo.focus=\\[Conformance\\] --minStartupPods=8",
```

Comment:
add `|\\[Feature:BootstrapTokens\\]|\\[Feature:NodeAuthorizer\\]` here as well

Reply:
Sure, thanks for review.